update.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -eo pipefail
+
+cd "$(dirname "$BASH_SOURCE")"
+
+versions=( "$@" )
+if [ ${#versions[@]} -eq 0 ]; then
+	versions=( */ )
+fi
+versions=( "${versions[@]%/}" )
+
+for version in "${versions[@]}"; do
+	md5="$(curl -fsSL --retry 5 "https://sourceforge.net/projects/geonetwork/files/GeoNetwork_opensource/v${version}/geonetwork.war.MD5/download")"
+
+	sed -ri \
+		-e 's/^(ENV GN_VERSION) .*/\1 '"$version"'/' \
+		-e 's/^(ENV GN_DOWNLOAD_MD5) .*/\1 '"$md5"'/' \
+		-e 's/^(FROM geonetwork):.*/\1:'"$version"'/' \
+		"$version/Dockerfile" "$version/postgres/Dockerfile"
+done
```

Comment:
The reason why I created the check against the *actual* online md5 rather than a hardcoded value, was because I realized that the war (and the checksum) had been re-generated for exactly the same (major-minor) version. The update script approach would obviously fail to address that, because we would be unaware that the file had changed and so we would not run the script...